lib/shoulda/matchers/integrations/registry.rb

```diff
@@ -26,3 +26,5 @@ def find_class!(name)
     end
   end
 end
+
+require 'shoulda/matchers/integrations/libraries'
```

Comment:
Perhaps this should remain in `shoulda/matchers/integrations`? If this is here then it won't get loaded until Registry is loaded.

Reply:
No problem! I'll do that!